nbformat/validator.py

```diff
@@ -230,7 +230,7 @@ def better_validation_error(error, version, version_minor):
 
 
 def validate(nbdict=None, ref=None, version=None, version_minor=None,
-             relax_add_props=False, nbjson=None):
+             relax_add_props=False, nbjson=None, repair=False):
```

Comment:
Let's make it `repair_dup_ids=True` to be more specific and better match default behavior today